core/src/main/java/nu/yona/server/subscriptions/service/BuddyInfoChangeMessageDto.java

```diff
@@ -109,7 +110,7 @@ public MessageActionDto handleAction(UserDto actingUser, Message messageEntity,
 			}
 		}
 
-		private MessageActionDto handleAction_Process(UserDto actingUser, BuddyInfoChangeMessage messageEntity,
+		private MessageActionDto handleAction_Process(User actingUser, BuddyInfoChangeMessage messageEntity,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS00100)